// Copyright 2018 The Chromium Authors
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.

// These messages are used to store a list of files and their digest. This is
// used to store the digests of loaded resources to validate them at runtime.

syntax = "proto2";

option optimize_for = LITE_RUNTIME;

package chrome_cleaner;

message FileDigest {
  optional string filename = 1;
  optional string digest = 2;
}

message FileDigests {
  repeated FileDigest file_digests = 1;
}
